pages/tours/index.tsx

```diff
@@ -143,15 +95,25 @@ export default function Tours() {
           </p>
         </Description>
         <ListCardsContainer>
-          {data.map((data, key) => (
-            <ListingCard key={key} data={data} />
-          ))}
+          {!state.isLoading && state.data ? (
+            state.data?.map((data, key) => (
+              <ListingCard key={key} data={data} />
+            ))
+          ) : (
+            <Loading />
```

Comment:
Remove the Load More button on loading. Show only once data were fetched

Reply:
Done.